tests/integration/hive_metastore/test_migrate.py

```diff
@@ -126,7 +126,7 @@ def test_migrate_tables_with_cache_should_not_create_table(
     assert target_tables[0]["tableName"] == table_name
 
 
-@retried(on=[NotFound], timeout=timedelta(minutes=2))
+@retried(on=[NotFound, PermissionDenied], timeout=timedelta(minutes=2))
```

Comment:
what message causes permission denied?

Reply:
I see this in the logs: 
databricks.sdk.errors.platform.BadRequest: PERMISSION_DENIED: Failed to get credentials: Storage Credential TEST_STORAGE_CREDENTIAL is not configured correctly. Please contact the owner or your account admin to update the configuration.